requirements.txt

```diff
@@ -47,21 +32,13 @@ pyOpenSSL>=0.11
 # Required by openstack.common libraries
 six>=1.9.0
 
-oslo.db>=1.7.0  # Apache-2.0
 oslo.i18n>=1.5.0  # Apache-2.0
 oslo.log>=1.0.0  # Apache-2.0
 oslo.messaging>=1.8.0  # Apache-2.0
```

Comment:
Ahh... osprofiler requires a notifier to be configured. So I think we do need to keep oslo-messaging and kombu.